library/alloc/src/alloc.rs

```diff
@@ -75,6 +75,7 @@ pub use std::alloc::Global;
 /// unsafe {
 ///     let layout = Layout::new::<u16>();
 ///     let ptr = alloc(layout);
+///     assert!(!ptr.is_null());
```

Comment:
You're usually supposed to call `alloc::alloc::handle_alloc_error` instead of panicking, aren't you?

Reply:
Fair.